tests/Unit/RunSingleTest/CMakeLists.txt

```diff
@@ -32,13 +32,14 @@ add_spectre_executable(
 
   # The following lines list the source files with tests in them to be
   # compiled into the executable.
-  ../DataStructures/Test_DataVector.cpp
+  ../NumericalAlgorithms/LinearSolver/Test_Gmres.cpp
```

Comment:
Did you mean to push the changes to this file?

Reply:
no, removed

Reply:
This still seems to be here.